packages/cli/src/workflows/workflows.controller.ee.ts

```diff
@@ -202,3 +186,163 @@ EEWorkflowController.post(
 		};
 	}),
 );
+
+EEWorkflowController.patch(
+	'/:id(\\d+)',
```

Comment:
`(\\d+)` is absent in the free version. Should it be in both?

Reply:
90% of this endpoint is identical to the free version.
- Should we abstract away the duplication and import it from both versions if the license allows?
- Should we mark the unique part of the paid version with `// enterprise-edition-start` and `// enterprise-edition-end` to be able to see this section more clearly if the license allows? I had to go through line by line side by side to find the differences.

If the license disallows this, we have a tension between maintainability and the license.